packages/repository-json-schema/test/integration/build-schema.integration.ts

```diff
@@ -527,57 +532,52 @@ describe('build-schema', () => {
         });
       });
     });
-
-    function expectValidJsonSchema(schema: JsonSchema) {
-      const ajv = new Ajv();
-      const validate = ajv.compile(
-        require('ajv/lib/refs/json-schema-draft-06.json'),
-      );
-      const isValid = validate(schema);
-      const result = isValid
-        ? 'JSON Schema is valid'
-        : ajv.errorsText(validate.errors!);
-      expect(result).to.equal('JSON Schema is valid');
-    }
   });
 
-  describe('getjsonSchema', () => {
-    it('gets cached JSON schema if one exists', () => {
-      @model()
-      class TestModel {
-        @property()
-        foo: number;
-      }
-      const cachedSchema: JsonSchema = {
-        properties: {
-          cachedProperty: {
-            type: 'string',
-          },
-        },
-      };
-      MetadataInspector.defineMetadata(
-        JSON_SCHEMA_KEY,
-        cachedSchema,
-        TestModel,
-      );
-
-      const jsonSchema = getJsonSchema(TestModel);
-      expect(jsonSchema).to.eql(cachedSchema);
-    });
+  function expectValidJsonSchema(schema: JsonSchema) {
+    const ajv = new Ajv();
+    const validate = ajv.compile(
+      require('ajv/lib/refs/json-schema-draft-06.json'),
+    );
+    const isValid = validate(schema);
+    const result = isValid
+      ? 'JSON Schema is valid'
+      : ajv.errorsText(validate.errors!);
+    expect(result).to.equal('JSON Schema is valid');
+  }
+});
 
-    it('creates JSON schema if one does not already exist', () => {
-      @model()
-      class NewModel {
-        @property()
-        newProperty: string;
-      }
+describe('getjsonSchema', () => {
```

Comment:
IIUC, this change is moving `getjsonSchema` test suite from `build-schema` suite to top-level. I'd rather avoid such unrelated change.